app/views/components/_timeout_modal.html.erb

```diff
@@ -0,0 +1,110 @@
+<div class="margin-y-3">
+  <a
+    id="button-toggle-timeout"
+    href="#session-timeout-modal"
+    class="hidden"
+    aria-controls="session-timeout-modal"
+    data-open-modal
+    ></a
+  >
+  <div
+    class="usa-modal"
+    id="session-timeout-modal"
+    aria-labelledby="modal-1-heading"
+    aria-describedby="modal-1-description"
+    data-force-action
+  >
+    <div class="usa-modal__content">
+      <div class="usa-modal__main">
+        <h2 class="usa-modal__heading" id="modal-1-heading">
+          Are you still using Touchpoints?
+        </h2>
+        <div class="usa-prose">
+          <p id="modal-1-description">
+            For security reasons,
+            your session will timeout in 5 minutes due to inactivity.
+            Do you want to extend your session?
+          </p>
+           <div class="timer" id="timer">5:00</div>
+        </div>
+        <div class="usa-modal__footer">
+          <ul class="usa-button-group">
+            <li class="usa-button-group__item">
+              <a
+                href="<%= admin_forms_path %>"
```

Comment:
When I press this button, it closes the modal but it also autoscrolls to the bottom of the page. Any idea why? It's weird behavior.

Reply:
good catch. will check it out